python/packages/ai/teams/ai/prompts/message.py

```diff
@@ -78,9 +78,9 @@ class ImageContentPart:
 Represents part of the message's content
 """
 
-
+@dataclass_json
```

Comment:
is there a reason you're updating this to a dataclass_json?

Reply:
Hi @singhk97 this is needed to deserialize OpenAI JSON response so that citations can be accessed via `response.message.context.citations`.